README.md

```diff
@@ -1,192 +1,125 @@
 # Temporal Java SDK Samples
 
-This repository contains sample Workflow applications that demonstrate various capabilities of Temporal using the [Java SDK](https://github.com/temporalio/sdk-java).
-
-- Temporal Server repo: https://github.com/temporalio/temporal
-- Temporal Java SDK repo: https://github.com/temporalio/sdk-java
-- Java SDK docs: https://docs.temporal.io/docs/java/introduction/
-
-## Table of Contents
-
-- [Temporal Java SDK Samples](#temporal-java-sdk-samples)
-  - [Table of Contents](#table-of-contents)
-  - [How to use](#how-to-use)
-  - [Temporal Web UI](#temporal-web-ui)
-  - [Running samples](#running-samples)
-  - [Samples directory](#samples-directory)
-    - [Hello samples](#hello-samples)
-    - [Scenario-based samples](#scenario-based-samples)
-    - [API demonstrations](#api-demonstrations)
-    - [SDK Metrics](#sdk-metrics)
-    - [Tracing Support](#tracing-support)
-  - [IDE Integration](#ide-integration)
-    - [IntelliJ](#intellij)
```

Comment:
Seems we lost the link to the server repo, the SDK repo, the docsm, and the TOC. Is this by intention?

Reply:
done